deployment/scripts/install.sh

```diff
@@ -286,22 +286,22 @@ fi
 echo "(Re-)Creating deployed objects"
 if [ "${KEEP_SUPPORT}" != true ]; then
 	oc process logging-support-template | oc delete -f - || :
-	oc delete serviceaccount,service,route --selector logging-infra=support
+	oc delete imagestream,service,route --selector logging-infra=support
```

Comment:
do we want to also delete image streams by default? won't that make dev builds more difficult?